src/sage/graphs/digraph.py

```diff
@@ -1719,29 +1717,28 @@ def feedback_edge_set(self, constraint_generation=True, value_only=False,
         ######################################
         # Ordering-based MILP Implementation #
         ######################################
-        else:
-            p = MixedIntegerLinearProgram(maximization=False, solver=solver)
+        p = MixedIntegerLinearProgram(maximization=False, solver=solver)
```

Comment:
This change does not seem right

Reply:
The block of code before is the method that generates constraints. It loops endlessly (`while True:`) until a solution is found and returned. So the code starting line 1720 is considered only when `constraint_generation` is `False`.

I will revert this change to avoid confusion.